docs/source/contribute/GSoC'21 Final Report.rst

```diff
@@ -11,111 +11,147 @@ Project Title: Integration of Alternate Code Analysis Tools
-ranging from license detection to copyright scanning and beyond in ScanCode.io in the form of
+The purpose of this project is to provide a way to implement and integrate alternate code analysis
+tools ranging from license detection to copyright scanning and beyond in ScanCode.io in the form of
 installable packages and wheels.
 
 As an example of this kind of integration, a Python wrapper package was built around
-`Google License Classifier v2 <https://pkg.go.dev/github.com/google/licenseclassifier/v2>`__
+`Google License Classifier v2`_
 to provide license detection as a new installable pipeline.
 
 Implementation
 --------------
 Over the course of this project, spanning over three months, we worked upon several
-key componenets. A high level overview of these components is provided below.
+key components. A high-level overview of these components is provided below.
 
 Converting Google License Classifier v2 to a C-style shared library
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-- One of the first goals of this project was to convert Google License Classifier to a form
-  that can be further converted to a **C-shared library**.
+- One of the main goals of this project was to convert Google License Classifier to a form
+  that can be further compiled into a **C-shared library**.
 
 - Since the License Classifier was written in **Golang**, there was native support for shared
   libraries. However, data marshaling and datatype conversion was required at several steps.
 
-- A **main package** file with **cgo bindings** had to be written for proper compilation.
+- A **main package** file with **cgo bindings** was written to achieve this objective.
 
-- Additionally, a **copyright notification** detection function was also ported from v1 package
+- Further, a **copyright notification** detection function was also ported from the **v1** package
   for additional functionality.
 
+- See `Google License Classifier with cgo bindings`_ for more details.
+
 Building Python wrapper and generating License Key Mappings
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 - A Python wrapper was built around the shared library to access all the underlying functionality
   using **ctypes**.
 
-- License key mappings, data marshaling, and error detection were handled by the package itself.
+- Data marshaling and error detection were handled by the package itself.
+
+- Further, license key mapping between License Classifier and ScanCode.io was generated and
+  integrated into the package. It helps in maintaining consistency between the two.
+
+- This Python package is available at `GoLicense-Classifier`_
 
 Developing a pipeline to run license detection
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- After developing the package, it was necessary to develop a pipeline that can be used to
+  establish communication with ScanCode.io.
```

Comment:
> establish communication with ScanCode.io
Is not great in that context.

I would suggest something along:
"After developing the package, it was necessary to integrate it into a pipeline that can be used to execute the License Classifier from the ScanCode.io app."